app/src/main/java/org/systers/mentorship/view/activities/FilterActivity.kt

```diff
@@ -0,0 +1,62 @@
+package org.systers.mentorship.view.activities
+
+import android.app.Activity
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import androidx.recyclerview.widget.LinearLayoutManager
+import kotlinx.android.synthetic.main.activity_filter.*
+import org.systers.mentorship.R
+import org.systers.mentorship.view.adapters.FilterAdapter
+import org.systers.mentorship.view.fragments.MembersFragment.Companion.filtersApplied
+
+class FilterActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_filter)
+
+        supportActionBar?.title=getString(R.string.filters)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+
+        rvFilters.apply {
+            layoutManager=LinearLayoutManager(context)
+            adapter=FilterAdapter(context)
+        }
+
+        btnApplyFilters.setOnClickListener {
+            /**
+             * Returning result to the Members Fragment
+             * */
```

Comment:
Thanks for adding the comments here :) But it's not required here as the function clearly states what it's responsible for. So I think you can remove this comments from here :)